content/library/videos/summer-2021/align-values-with-job.md

```diff
@@ -20,6 +20,23 @@ duration: 25
 cover: >-
   https://i.vimeocdn.com/video/1349709776-7f64dd2de2dc22bee68b7b241346ad8edfe5514c2ca2454e6ed3271ef4e8ce81-d
 ---
+<extra-reading title="Summary">
+<!-- There is am empty line required above and below content. -->
+
+While motivation and acquiring discipline are important to do our jobs well, the secret to remaining aligned with what we do is by keeping the inner child spirit alive.
+- Step 1: Know your values
```

Comment:
The formatting of these looks weird. Can you make it one line with the content, and use a `<br>` tag to break line?